README.md

```diff
@@ -1 +1,218 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+<details>
+  <summary>Click to view contents</summary>
+  <ol>
+    <li>
+      <a href="#about-project">📖 About the Project</a>
+        <ul>
+          <li><a href="#video">📹 Video</a></li>
+          <li>
+            <a href="#built-with">🛠 Built With</a>
+            <ul>
+              <li><a href="#tech-stack">Tech Stack</a></li>
+              <li><a href="#key-features">Key Features</a></li>
+            </ul>
+          </li>
+          <li><a href="#live-demo">🚀 Live Demo</a></li>
+        </ul>
+    </li>
+    <li>
+      <a href="#getting-started">💻 Getting Started</a>
+      <ul>
+        <li><a href="#setup">Setup</a></li>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#install">Install</a></li>
+        <li><a href="#usage">Usage</a></li>
+        <li><a href="#run-tests">Run tests</a></li>
+        <li><a href="#deployment">Deployment</a></li>
+      </ul>
+    </li>
+    <li><a href="#authors">👥 Authors</a></li>
+    <li><a href="#future-features">🔭 Future Features</a></li>
+    <li><a href="#contributing">🤝 Contributing</a></li>
+    <li><a href="#support">⭐️ Show your support</a></li>
+    <li><a href="#acknowledgements">🙏 Acknowledgements</a></li>
+    <li><a href="#faq">❓ FAQ</a></li>
+    <li><a href="#license">📝 License</a></li>
+  </ol>
+</details>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+This **Budget App** is a ruby on rails application that allows users to create and update posts.
+
+## 📹 Video <a name="video"></a>
+
+<div align="center">
+  <img width="700" src="recipe-app.gif" />
+</div>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Rails</a></li>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+    <li><a href="https://javascript.com/">Javascript</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+This projects aim is to help users implement a real-world project that
+
+- where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Video Demo Link](https://www.loom.com/share/ca59940837a04b618b95d787c663d174)
+
+- [Live Demo Link]()
```

Comment:
- [ ] Kindly deploy your app and add the deployment link in the readme file, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=is%20tested%20adequately.-,Deploy%20the%20project%2C,-and%20test%20for) requirement.

Reply:
Hi @ShadyShawkat 
![2022-12-23 (3)](https://user-images.githubusercontent.com/96915150/209366530-fb5036b1-da48-449f-8d02-45e9aabbe95d.png)

Thanks for that observation, Micoverse asked us to deploy to Heroku but unfortunately, Heroku does not support free web hosting anymore! This began on Nov 28 this year. For now, the most important thing is that my app is working as you can see from my loom video. On my own, I will search for another free rail web hosting site, learn how to deploy using their platform, and deploy it later. Kindly approve the project.
Thank you!